examples/caltrain/caltrain_text_mode.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>nit: "LlamaParse markdown mode doesn't quite work yet with gpt-4o - it's in development!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>